packages/pn-personagiuridica-webapp/src/App.tsx

```diff
@@ -44,85 +51,111 @@ const productsList: Array<ProductSwitchItem> = [
   const dispatch = useAppDispatch();
-  const { t, i18n } = useTranslation(['common']);
+  const { t, i18n } = useTranslation(['common', 'notifiche']);
+  const loggedUser = useAppSelector((state: RootState) => state.userState.user);
+  const { tos, fetchedTos } = useAppSelector((state: RootState) => state.userState);
+  const currentStatus = useAppSelector((state: RootState) => state.appStatus.currentStatus);
   const { pathname } = useLocation();
   const path = pathname.split('/');
   const source = path[path.length - 1];
 
-  const currentStatus = useAppSelector((state: RootState) => state.appStatus.currentStatus);
+  const sessionToken = loggedUser.sessionToken;
+  const jwtUser = useMemo(
+    () => ({
+      id: loggedUser.fiscal_number,
+      name: loggedUser.name,
+      surname: loggedUser.family_name,
+      mail: loggedUser.email,
+    }),
+    [loggedUser]
+  );
 
-  // TODO: remove mocked data
-  const jwtUser = {
-    id: 'FiscalCode',
-    name: 'Name',
-    surname: 'Surname',
-    mail: 'mail',
-  };
+  const isPrivacyPage = path[1] === 'privacy-tos';
 
-  // TODO: define side menu items
-  const menuItems = useMemo(() => {
-    const basicMenuItems: Array<SideMenuItem> = [
-      { label: 'menu.notifications', icon: Email, route: routes.NOTIFICHE },
-      // TODO: gestire badge
-      {
-        label: 'menu.delegations',
-        icon: AltRouteIcon,
-        route: routes.DELEGHE,
-        rightBadgeNotification: undefined,
-      },
-      { 
-        label: 'menu.app-status', 
-        // ATTENTION - a similar logic to choose the icon and its color is implemented in AppStatusBar (in pn-commons)
-        icon: () => currentStatus 
-          ? (currentStatus.appIsFullyOperative
-            ? <CheckCircleIcon sx={{ color: 'success.main' }} />
-            : <ErrorIcon sx={{ color: 'error.main' }} />)
-          : <HelpIcon />
-        , 
-        route: routes.APP_STATUS 
-      },
-    ];
-    const items = { ...getMenuItems(basicMenuItems, 'id-organizazzione') };
-    // localize menu items
-    /* eslint-disable-next-line functional/immutable-data */
-    items.menuItems = items.menuItems.map((item) => ({ ...item, label: t(item.label) }));
-    if (items.selfCareItems) {
-      /* eslint-disable-next-line functional/immutable-data */
-      items.selfCareItems = items.selfCareItems.map((item) => ({ ...item, label: t(item.label) }));
+  useUnload(() => {
+    trackEventByType(TrackEventType.APP_UNLOAD);
+  });
+
+  useTracking(MIXPANEL_TOKEN, process.env.NODE_ENV);
+
+  useEffect(() => {
+    // init localization
+    initLocalization((namespace, path, data) => t(path, { ns: namespace, ...data }));
+    // eslint-disable-next-line functional/immutable-data
+    errorFactoryManager.factory = new PFAppErrorFactory((path, ns) => t(path, { ns }));
+  }, []);
+
+  useEffect(() => {
+    if (sessionToken !== '') {
+      void dispatch(getDomicileInfo());
+      void dispatch(getSidemenuInformation());
```

Comment:
This dispatch call Delegate api. Comment it out

Reply:
ok